src/overlays/actors/ovl_En_Zl4/z_en_zl4.h

```diff
@@ -9,9 +9,13 @@ typedef void (*EnZl4ActionFunc)(struct EnZl4*, PlayState*);
 
 typedef struct EnZl4 {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x14C];
+    /* 0x144 */ SkelAnime unk144;
+    /* 0x188 */ char pad188[0x108];
     /* 0x290 */ EnZl4ActionFunc actionFunc;
-    /* 0x294 */ char unk_294[0x60];
-} EnZl4; // size = 0x2F4
+    /* 0x294 */ char pad294[0x4C];
+    /* 0x2E0 */ s16 unk2E0;
+    /* 0x2E2 */ char pad2E2[0xE];
+    /* 0x2F0 */ s32 unk2F0;
```

Comment:
```suggestion
    /* 0x2F0 */ s32 envColorAlpha;
```
since `func_809A1E28` and `func_809A1DD0` treat it as such?